src/Symfony/Component/Notifier/Bridge/Telegram/Tests/TelegramTransportTest.php

```diff
@@ -1242,6 +1242,6 @@ public function testUsingMultipleExclusiveOptionsWillProvideExceptions(TelegramO
         $transport = self::createTransport($client, 'testChannel');
 
         $this->expectException(MultipleExclusiveOptionsUsedException::class);
```

Comment:
We have a test here, so it should have failed 🤔

Reply:
And I can see it failing in `8.2, high deps`:
```
1) Symfony\Component\Notifier\Bridge\Telegram\Tests\TelegramTransportTest::testUsingMultipleExclusiveOptionsWillProvideExceptions with data set "edit" (Symfony\Component\Notifier\Bridge\Telegram\TelegramOptions Object (...))
PHPUnit\Framework\Exception: Class "Symfony\Component\Notifier\Exception\MultipleExclusiveOptionsUsedException" does not exist
```

But this shouldn't fail anymore, as I raised symfony/notifier to ^6.4 🤔

Reply:
for high-deps, this will install the 7.0 component. Is the other PR merged up already ?

the high-deps job has this weird situation for the time were we have 2 active development branches